History.markdown

```diff
@@ -31,71 +67,22 @@
+  * chore(ci): add workflow to build and profile third-party repo (#8398)
+  * chore(ci): use Ruby 2.7.1 in GitHub Actions (#8444)
+  * chore(ci): fix test suite compatibility with JRuby (#8418)
   * chore(deps): bump Rubocop to 0.93.0 (#8430)
-  * Use Ruby 2.7.1 in GitHub Actions (#8444)
-  * Reduce allocations from rendering item as liquid (#8406)
-  * Test that Liquid expressions are not deeply evaled (#8292)
-  * Test rendering arbitrary Liquid variables by default (#7414)
+  * chore(deps): cucumber gem version to 5.1.2 (#8413)
+  * fix: `rake console` generating LoadError (#8312)
+  * style: configure Performance cops (#8369)
+  * style: RuboCop to-do file (#8296)
+  * style: enable `Performance/ChainArrayAllocation` cop (#8404)
+  * style: enable Lint/NoReturnInBeginEndBlocks Cop (#8457)
 
-### Bug Fixes
-
-  * Replace nested conditional with guard clauses (#8294)
-  * fix: security bump (#8349)
-  * Fix path matching regex in post_url Liquid tag (#8375)
-  * Reduce string allocations from the `link` tag (#8387)
-  * Optimize parsing of parameters in include tag (#8192)
-  * Stash documents `write?` attribute in a variable (#8389)
-  * Reduce string allocations from generating doc URLs (#8392)
-  * Check if site is in incremental mode optimally (#8401)
-  * Handle nil argument to Jekyll.sanitized_path (#8415)
-  * Compute relative_path of pages using PathManager (#8408)
-  * Enable `Performance/ChainArrayAllocation` cop (#8404)
-  * Utilize flexibility of Site#in_dest_dir (#8403)
-  * Reduce allocation from normalize_whitespace filter (#8400)
-  * Use Regexp#match? when MatchData is not required (#8427)
-  * Check default front matter scope against symbols (#8393)
-  * Stash frequently used Drop setter keys for reuse (#8394)
-  * Memoize defaults computed for Convertibles (#8451)
-  * Reduce array allocations from merging categories (#8453)
-  * Enable Lint/NoReturnInBeginEndBlocks Cop (#8457)
-  * Memoize destination of pages, documents and staticfiles (#8458)
-  * Generate items from `site.include` list only once (#8463)
-  * Allow triggering `:post_convert` events atomically (#8465)
-
-### Minor Enhancements
-
-  * Warn on command-line with permalink conflict (#8342)
-  * Supress warning issued for redirect pages (#8347)
-  * Add post_convert hook to modify HTML content before layout (#8368)
-  * Cache Jekyll.sanitized_path (#8424)
-  * Memoize array of drop getter method names (#8421)
-  * Enhance detection of conflicting destination URLs (#8459)
-  * Debug reading Page and Layout objects (#8100)
-  * Don&#39;t reset site.url to localhost:4000 by default (#7253)
+  * test: rendering arbitrary Liquid variables by default (#7414)
+  * test: that Liquid expressions are not deeply evaled (#8292)
 
 ## 4.1.1 / 2020-06-24
 
-  * Fix grammar in documentation section (#8265)
-
-### Development Fixes
-
-  * Bump RuboCop to v0.85.x (#8223)
-  * Expect drive letter only on vanilla windows (#8227)
-
```

Comment:
Lets not include changes to this section in the pull request.
It can be pushed to `master` directly.